decoder/main.go

```diff
@@ -252,18 +258,20 @@ func UpdateFortBatch(ctx context.Context, db db.DbDetails, p []RawFortData) {
 }
 
 func UpdatePokemonBatch(ctx context.Context, db db.DbDetails, wildPokemonList []RawWildPokemonData, nearbyPokemonList []RawNearbyPokemonData, mapPokemonList []RawMapPokemonData, username string) {
-	if config.Config.Tuning.ProcessWilds {
-		for _, wild := range wildPokemonList {
-			encounterId := strconv.FormatUint(wild.Data.EncounterId, 10)
-			pokemonMutex, _ := pokemonStripedMutex.GetLock(encounterId)
-			pokemonMutex.Lock()
+	for _, wild := range wildPokemonList {
+		encounterId := strconv.FormatUint(wild.Data.EncounterId, 10)
+		pokemonMutex, _ := pokemonStripedMutex.GetLock(encounterId)
+		pokemonMutex.Lock()
+
+		spawnpointUpdateFromWild(ctx, db, wild.Data, int64(wild.Timestamp))
 
+		if config.Config.Tuning.ProcessWilds {
 			pokemon, err := getOrCreatePokemonRecord(ctx, db, encounterId)
 			if err != nil {
 				log.Errorf("getOrCreatePokemonRecord: %s", err)
 			} else {
 				if pokemon == nil || pokemon.isNewRecord() || pokemon.wildSignificantUpdate(wild.Data) {
```

Comment:
pokemon can not be `nil` anymore here :)
`getOrCreatePokemonRecord` returns error or a pokemon object (not nullable)